common/data_refinery_common/models/models.py

```diff
@@ -1524,13 +1523,6 @@ def terms_and_conditions(self):
         return settings.TERMS_AND_CONDITIONS
 
 
-"""
-# Associations
-
-These represent the relationships between items in the other tables.
-"""
-
-
```

Comment:
I don't like these changes. These docstring denoted different parts of the file.

We should probably handle this issue by breaking this file up into different namespaces, but just moving all these docstrings to the top doesn't really make much sense.

Reply:
yeah, I wasn't sure what to do with that rule. I'll revert these and comment it until we simplify `models.py`